PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+(Describe your pull request in 1-2 sentences)
+
+Fix # (optional)
+
+Rationale and detailed description of the changes:
+
+-
+-
+-
+
+Tasks before review:
+
+- [ ] Add tests and examples of any added functionality
+    - [ ] For bugfix pull requests, list any previously-disabled tests that are
```

Comment:
by disabled you mean commented out or is there a mechanism to mark tests as disabled?

Reply:
Commented out or otherwise disabled (e.g. by setting some variables like I did for the finite-difference gradient tests on periodic structures). I didn't want to specify the mechanism, since there might be several (including not having written the test in the first place, but that's another can of worms)

Reply:
Ok.

Is there a requirement that bugfix PR should come with some additional tests to prevent regression?

Reply:
No, but it's heavily implied by the second part of this bullet point.